FranzKrekeler/submission_and_tutorial.ipynb

```diff
@@ -0,0 +1,608 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Why did you take the CumSum of the returns? Thats wrong, it should be the cumproduct, otherwise, you are not compounding your returns.

`sp500_prices[sample_partners].pct_change(fill_method='ffill').dropna(how='all').cumsum(axis=0).plot();`